src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
It's not a good practice to have multiple sources of truth but in this case `BASE_URL` is defined several times. Try to create separate file for your const's.